lms/envs/common.py

```diff
@@ -3933,3 +3933,6 @@ def _make_locale_paths(settings):  # pylint: disable=missing-function-docstring
 
 # Notification email settings
 NOTIFICATION_FROM_EMAIL = 'info@omnipreneurshipacademy.com'
+
+# CDN link
```

Comment:
Can we better describe this, like what is this CDN link used for, the way it is described in line 487?